src/components/Book.jsx

```diff
@@ -6,12 +6,16 @@ import { removeBook } from '../redux/books/books';
 const Book = ({ title, author, id }) => {
   const dispatch = useDispatch();
   return (
-    <div className="flex w-[500px] justify-between items-center border-2 border-white/30 p-2 rounded-lg shadow-md">
-      <div>
+    <div className="flex my-8 mx-8 justify-between items-center border-2 border-white/30 p-2 rounded-lg shadow-md">
+      <div className="flex flex-col">
         <h4 className="text-2xl">{title}</h4>
-        <p>{author}</p>
+        <p className="text-[#0290ff]">{author}</p>
+        <div className="py-2">
+          <button type="button" className="text-[#0290ff] pr-2 cursor-pointer ">Comments</button>
+          <button onClick={() => dispatch(removeBook(id))} type="button" className="text-[#0290ff] pr-2 cursor-pointer ">Remove</button>
+          <button type="button" className="text-[#0290ff] pr-2 cursor-pointer ">Edit</button>
+        </div>
       </div>
-      <button onClick={() => dispatch(removeBook(id))} type="button" className="border-2 border-fuchsia-500 p-2 cursor-pointer rounded-lg">Remove</button>
     </div>
   );
 };
```

Comment:
- [ ] We are required to style the app as it is on the design exactly. Kindly take note of things like font weight, font size, font family and so on.

- [ ] You are also missing some elements like the user icon on the top right, the elements on the right side of the book components. Kindly try to replicate the design exactly.

  Your Implementation | Zeplin Design
  :---:|:---:
  ![urit](https://user-images.githubusercontent.com/54848363/214617633-b174ec04-6390-4802-99fe-f774801825cb.jpg) | ![die](https://user-images.githubusercontent.com/54848363/214617478-5132f60d-ab49-479b-a33c-7adf1a5a2130.jpg)

Reply:
Thank you @DammyShittu I have fixed these issues.